assets/section-main-product.css

```diff
@@ -298,6 +298,10 @@ fieldset.product-form__input .form__label {
 
 /* Product info */
 
+.different-color-scheme {
+  padding: 2rem;
+}
+
```

Comment:
Another spacing fix I think to do is removing the top margin on the first element so that there isn't extra spacing here: 
![](https://screenshot.click/12-19-o29fq-80dzq.png)